scripts/devops_tasks/common_tasks.py

```diff
@@ -249,4 +288,51 @@ def is_error_code_5_allowed(target_pkg, pkg_name):
         ):
         return True
     else:
-        return False
\ No newline at end of file
+        return False
+
+# This function parses requirement and return package name and specifier
+def parse_require(req):
+    req_object = Requirement.parse(req)
+    pkg_name = req_object.key
+    spec = SpecifierSet(str(req_object).replace(pkg_name, ""))
+    return [pkg_name, spec]
+
+
+# This method installs package from a pre-built whl
+def install_package_from_whl(package_name, whl_directory, working_dir, python_sym_link = sys.executable):
+    if not os.path.exists(whl_directory):
+        logging.error("Whl directory is incorrect")
+        exit(1)
+
+    logging.info("Searching whl for package {}".format(package_name))
+    whl_name = "{}*.whl".format(package_name.replace("-", "_"))
```

Comment:
This eliminates a bunch of the specificity that we just encoded right? (Handling python versioning, etc)